test_vectors/test_bls.yml

```diff
@@ -0,0 +1,381 @@
+- input: ['0x961a0878c110dc3d12d39315369840878021a179b32e7660b82a7e514ba26a7b2a75030e4921fada1e3a3dbde2a7db0700ace55e6b17e110be740f928e26e57ce1b7162dd2c5776c39fd04de07797e60fe1eda3b43eeaa480b701ab371270c02',
+    '0x8b9e527355eb4ddc2b9bd346b671d44d701486027700504d001491de262667236e7ed6e0b60fc048057ba8d12be5f1841470a27f67754357bc412c005aa0085461d40bf8302d199390367947d331cd93c7d03ab826fbb214872dc65ca2b3b654',
+    '0x97a5fd716644fbbf2abc6c21aca80f6876c55f78128f8208959277e9f9a108a3438938200f873dfb21be2e1230ba720209a1d0e366089ccf40802b56e0e417d888a10c92e699ff60927e17e6cf6fc01d4a3ccf48b26cfdd1cf37fa47bf2ebde0']
+  output: '0x139a625c3f9aab07af282aa139e0ec87f61dbd1a1368820408c9ba16f6cb67e5eebe41f2394c9fd759dbf861b9d0c6040e0510c1844b287fcef9e25c9e441a8ee7a1812bd22c9a3a401ced74d784e1c7fd2baf9404f1c7ed88f303653186cf1b'
+- input: ['0x86a3bf976c0c61b667903ac2890ad80502d31cc8c894de6b2a51ec0c418e85922abe3099e831f5223b5a89d8348f2c500fa567a000579ffbbadde8aea769a4f88a6f70df419bb58fb8c494092e79bbfa2ab42119c5f9a6ea530b4da0ea98d647',
+    '0x8d3259033ab587f06840b71dd29e258d889c6c54416a680e0c9fb58deaf4eab2235165ebc0d3cb31479dfef75710ee4511ef457c45216cf7e526d34f484652db5e6a509d58a284669e8a60161efa15e06ce521847d3551f0ecbcbf64b193a264',
+    '0x18c78d38f2dc9d7cf492c8dc0d071275c7b1079c46d0c31e6b83e3cba9bacc49bd87145eef2a0f62dfd7f38289b400fd15420a57521a6ff5fcb5808d17805da8e5e2f34f590d74f23c85b48413dd5c6540faf30995234b858b1d651221ede33f']
+  output: '0x016063cb6fc5348554c04dda15f226459ce031f3e36cfb13caa57e42ec73829f5e60f53a822f0d8115ee52ad5e66a0311380d3f019d291b17b330c6b877f104e7d2d9a8824c1e43c8e0e23f8af585c38dbb9abaf7a1d8b77964d8806b764f250'
+- input: ['0x18982989a52dbca77e4cb555feef40e8b84d4bc7644be428006e696831d41a0ece4bb367c708d8cb432675ab24e9118912a1e575c26fe3174c1db95e76563a812789ae7d2e271cfcc1aecab2e2dcefd87f5089d532a71cfe112cf6d8b3d3d558',
+    '0x10599c197b1a7a15ea9b6d668bc4c6591e83425848f0fcf0f623007aa48f8182cbd745df06b335a20505a1f8044b788208612d748cc9c5e71470ad8115284a249519bb7c650bf19ecdc7a1de3b6872fa806d0038a478af081119d90d70fe55ea',
+    '0x14f291cb1bef5ddc18a69f00f439bce9a1e9fc95f65ca0b00578f1d12bf14ec8b51046feb3b144c2522b3310677019c40f29f2adf5401f68f065201aac54387bb4adecb923a4fcb207115c8cc42590598046c26b5544f090b1ecee5aa0501193']
+  output: '0x8d9788b7fa3a51d83449f0d49266b0f63f236185b024d193b3dde2345c14bc42f73fd788fa18b2226de52e9af241349c16c40bc4df29df320abdf0d950c0f43fbb311bf7a34e1ee82f568071822be76d9c65998a04bfc57b394f89dbfb9ff249'
+case07_aggregate_pubkeys:
+  input: ['0x8c5275be9b8e1c4165368c53b86159c868ab4d8074d0311aece42ea17d7b0d5d3e6715fb6b64ec7832d26efccef7f0e8',
+    '0x970e680ff06f59de5584c2bbc1952652cb98bf23d0e5f5db50f527795e0c14404cd79f43048b54e7d9d2672c77b5ce6c',
+    '0x13286c99785f02ce2ad36cc70fbff5705e433d172ab35808502257a3c0dbb0e2172a76a64c8bcc2352c60d4034434add']
+  output: '0x000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000'
```

Comment:
This is 0 because I forgot a `pubkeys.append(pubkey)` in ̀case03_private_to_public_key`

Reply:
Fixed. Thank you for the review